### PR TITLE
Italian versions of Name, Address, Color and Internet

### PR DESCRIPTION
**Pull request by pmontrasio**
I've added:

- lib/faker/name/it.ex
- lib/faker/address/it.ex
- lib/faker/color/it.ex
- lib/faker/internet/it.ex
- links in USAGE.md to the Italian modules

See my comment to issue 123 about how I made doctests pass in my code https://github.com/igas/faker/issues/123#issuecomment-438328161

### CODE REVIEW

File: lib/faker/name/it.ex [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+      iex> Faker.Name.It.first_name()
+      "Azalea"
+      iex> Faker.Name.It.first_name()
+      "Dionigi"
+      iex> Faker.Name.It.first_name()
+      "Agave"
+  """
+  @spec first_name() :: String.t()
+  sampler(:first_name, [
+        "Abramo", "Acantha", "Ace", "Achille", "Ada", "Adamo",
```
Comment (vbrazo) [inline lib/faker/name/it.ex]: You should run the formatter for this file to keep consistency:`mix format lib/faker/name/it.ex`

**Comment by vbrazo** [comment]
You should also update the `changelog`.

File: lib/faker/name/it.ex (hunk 1 above)
Comment (pmontrasio) [reply lib/faker/name/it.ex]: I was sure I had the formatter running on save like in other projects but it didn't run. Fixed it.
I updated the Changelog. Please check that my update is in the right place.

File: lib/faker/util.ex [hunk 2]
```diff
@@ -20,14 +20,22 @@ defmodule Faker.Util do
...
+      {"state1", "city1"}
   """
-  @spec pick(Enum.t()) :: any
-  def pick(enum) do
-    enum
-    |> Enum.to_list()
-    |> Enum.at(Faker.random_between(0, Enum.count(enum) - 1))
+  @spec pick([any]) :: any
+  def pick(list) do
+    Enum.at(list, Faker.random_between(0, Enum.count(list) - 1))
```
Comment (vbrazo) [inline lib/faker/util.ex]: This perfomance issue was already fixed https://github.com/igas/faker/pull/221

Comment (pmontrasio) [reply lib/faker/util.ex]: Sorry, I didn't realize that if I added more commits to master on my fork after the approval those commits would still be included in the pull request.
What should I do now? Maybe I'll start again from 0.11.2, make a branch, add the Italian modules, create a new pull request from there and forget about this one. Would that be ok?

Comment (vbrazo) [reply lib/faker/util.ex]: Just update this branch with master and we'd be okay.

Comment (vbrazo) [reply lib/faker/util.ex]: In other words, fix the merge issues.

Comment (pmontrasio) [reply lib/faker/util.ex]: Done.

File: CHANGELOG.md [hunk 3]
```diff
@@ -28,6 +28,10 @@ Change log itself follows [Keep a CHANGELOG](http://keepachangelog.com) format.
 
 ## 0.11.1
 
+### Added
+
+* `Faker.Address.It`, `Faker.Color.It`, `Faker.Internet.It` and `Faker.Name.It` [[@pmontrasio][]]
+
```
Comment (vbrazo) [inline CHANGELOG.md]: These changes should be in the `unreleased` second.

Comment (pmontrasio) [reply CHANGELOG.md]: Done.

**Comment by igas** [comment]
Thank you @pmontrasio!

**Pull request merged**